src/RazorSdk/SourceGenerators/RazorSourceGenerationContext.cs

```diff
@@ -136,17 +136,9 @@ private static (IReadOnlyList<RazorInputItem> razorFiles, IReadOnlyList<RazorInp
 
                 options.TryGetValue("build_metadata.AdditionalFiles.CssScope", out var cssScope);
 
-                if (!options.TryGetValue("build_metadata.AdditionalFiles.GeneratedOutputFullPath", out var generatedOutputPath))
-                {
-                    context.ReportDiagnostic(Diagnostic.Create(
-                        RazorDiagnostics.GeneratedOutputFullPathNotProvided,
```

Comment:
This too goes away?

Reply:
I'll do this in a separate PR. The diagnostics file actually needs a general cleanup overall.